docs/home/index.md

```diff
@@ -70,13 +63,11 @@ scalability of a distributed architecture.
 :padding: 2
 :class-title: sd-fs-5
 
-Connect your applications to CrateDB and CrateDB Cloud, using database
-drivers, libraries, adapters, connectors, and dialects.
+Learn how to connect your applications using database drivers, libraries,
+adapters, and connectors.
 
 CrateDB supports both the [HTTP protocol] and the [PostgreSQL wire protocol],
-which ensures that many clients that work with PostgreSQL, will also work with
-CrateDB. Through corresponding drivers, CrateDB is compatible with [ODBC],
-[JDBC], and other database API specifications.
+ensuring compatibility with many PostgreSQL clients.
```

Comment:
I wanted to do a bit of name dropping around JDBC and ODBC here, for feeding search engine crawlers. Is there a chance to keep those while shortening the text(s)?

Reply:
Re-added with b15208feb.